include/engine/guidance/assemble_steps.hpp

```diff
@@ -109,6 +116,7 @@ inline std::vector<RouteStep> assembleSteps(const datafacade::BaseDataFacade &fa
                                           NO_ROTARY_NAME,
                                           segment_duration / 10.0,
                                           distance,
```

Comment:
@MoKob the reasons are lost in the archives of history, but I believe the thinking was "what's the smallest unit we would want to keep" and "let's store it as an integer".  Combine these ideas, and you get "meters" and "deciseconds" 😛